Marlin/Configuration_adv.h

```diff
@@ -578,6 +578,7 @@
 //#define BEZIER_CURVE_SUPPORT
 
 // G38.2 and G38.3 Probe Target
+// enable PROBE_DOUBLE_TOUCH if you want G38 to double touch
```

Comment:
Capitalize "Enable" here. And add this line to all other `Configuration_adv.h` files.
Search:
```
// G38.2 and G38.3 Probe Target
//#define G38_PROBE_TARGET
```
Replace:
```
// G38.2 and G38.3 Probe Target
// Enable PROBE_DOUBLE_TOUCH if you want G38 to double touch
//#define G38_PROBE_TARGET
```